cpyamf/codec.pyx

```diff
@@ -198,6 +198,26 @@ cdef class IndexedCollection(object):
         return n
 
 
+cdef class ByteStringReferenceCollection(IndexedCollection):
+    """
+    There have been rare hash collisions within a single AMF payload causing
+    corrupt payloads.
+
+    Which strings cause collisions is dependent on the python runtime, each
+    platform might have a slightly different implementation which means that
+    testing is extremely difficult.
+    """
```

Comment:
No - since they are accessing the same properties within IndexedCollection (and nothing in ByteStringReferenceCollection)